### DISCUSSION TITLE
Codesandbox Black 2021 theme available on VSCode

### DISCUSSION THREAD

**Discussion by jvzaniolo** [codesandbox/codesandbox-client#6073] [General] [answered]
Hey guys, I loved the new dark theme! Congratulations!
Could you make it available on VSCode through an extension? I would love to use the theme there too.

**Comment by smhutch** [comment, marked as answer]
@jvzaniolo Awesome, glad you like it! 🖤

We don't have a timeline yet, but we do plan to share a VSCode extension that will allow you to use the theme in your editor 😄

**Reply by jvzaniolo** [reply]
@smhutch Awesome, thank you!
